#!/usr/bin/python3



def raise_exception_msg(message=""):

    """Raise a NameError exceeption with a message."""

    raise NameError(message)
